src/tool/python/file_writers.h

```diff
@@ -106,46 +99,59 @@ namespace xlang
     {
         writer w;
 
+        write_license_python(w);
         w.write(strings::setup, module_name, native_module_name, bind<write_setup_filenames>(native_module_name, namespaces));
         create_directories(folder);
         w.flush_to_file(folder / "setup.py");
     }
 
-    inline void write_package_init(stdfs::path const& folder, std::string_view const& module_name)
+    inline void write_package_dunder_init_py(stdfs::path const& folder, std::string_view const& module_name)
```

Comment:
>dunder [](start = 30, length = 6)

dunder?

Reply:
"dunder <name>" is how you pronounce `__name__` (rather than "double underscore name double underscore").

It's different from "double underscore <name>", which is `__name`.

Reply:
Double UNDERscore https://wiki.python.org/moin/DunderAlias